WDL/runtime/__init__.py

```diff
@@ -14,6 +14,7 @@ def run(
     cfg: config.Loader,
     exe: Union[Tree.Task, Tree.Workflow],
     inputs: Env.Bindings[Value.Base],
+    wdl_doc: Document,
```

Comment:
There is a helper function to traverse the AST and add a "parent" attribute to each object, so you could e.g. get from a `Task` object to the containing `Document` without having to pass it here. However, this transformation isn't applied by default ([example application](https://github.com/chanzuckerberg/miniwdl/blob/6c68acf552e947a2d0a2da32b0cad4f9d8b8e72c/examples/paste_wdl_imports.py#L46-L47), [usage](https://github.com/chanzuckerberg/miniwdl/blob/6c68acf552e947a2d0a2da32b0cad4f9d8b8e72c/examples/paste_wdl_imports.py#L105-L106)). Probably these links are so frequently useful that we should just make them available always. Let me try that on a branch and see if something blows up (it probably will >.<)

Reply:
OK -- if you merge 835ff0e from master, you should be able to just `getattr(exe, 'parent')` to get the containing `Document`!

Reply:
this is great! thank you